pkg/config/con.go

```
@@ -0,0 +1,922 @@
+			continue
+		}
+		if found {
+			cl.configFound = true
+			cl.AtmosConfigPaths = append(cl.AtmosConfigPaths, configPath)
+		}
+	}
+	return nil
+}
+
+// findGitTopLevel finds the top-level directory of a Git repository
+func findGitTopLevel(startDir string) (string, error) {
```

golangci-lint (1.62.2)

func `findGitTopLevel` is unused

(unused)